src/main/java/org/dormitory/autobotsoccub/command/txtcommand/HelloCommand.java

```diff
@@ -4,22 +4,24 @@
 import org.dormitory.autobotsoccub.command.TextCommand;
 import org.dormitory.autobotsoccub.command.keyboard.KeyboardFactory;
 import org.dormitory.autobotsoccub.command.result.CommandResult;
+import org.dormitory.autobotsoccub.user.UserUtils;
 import org.telegram.telegrambots.api.objects.Message;
 import org.telegram.telegrambots.api.objects.Update;
-import org.telegram.telegrambots.api.objects.User;
 
 import java.util.Optional;
 import java.util.regex.Pattern;
 
-import static org.dormitory.autobotsoccub.command.keyboard.Button.REGISTER;
+import static org.dormitory.autobotsoccub.command.keyboard.Button.*;
 
 @AllArgsConstructor
 public class HelloCommand implements TextCommand {
 
     private static final Pattern CMD_PATTERN = Pattern.compile("/hello");
-    private static final String REPLY = "Hello, %s!";
+    private static final String REPLY = "Hello, %s";
     private static final String UNKNOWN_USER = "anonymous";
 
+    private static final UserUtils USER_UTILS = UserUtils.getINSTANCE();
```

Comment:
Say "no" to singleton. Use either spring bean with corresponding scope, which is default, or simply static class. Later is preferred in given case.

Reply:
I say: "No, singleton!", and he left) Now used static.